### PR TITLE
Arcana Auth SDK support

### PR DESCRIPTION
**Pull request by mmjee**
### Description
Added a package to support the Arcana Auth SDK. See the included documentation for more information.

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [x] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [x] Add/update the related package in the `docs/package.json` file (if applicable)

Added documentation for the new package.

Documentation screenshot

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected) – Not relevant
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

#### Tests with demo app (SDK)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect – Requires some changes right now

### CODE REVIEW

**Comment by mmjee** [comment]
So, how is the documentation package supposed to install a unpublished package? Should I split this into two PRs, so that the package can be published first?

File: packages/demo/package.json
Comment (leightkt) [inline packages/demo/package.json:27]:
```suggestion
    "@web3-onboard/arcana-auth": "^2.0.0-alpha.1",
```

File: docs/package.json
Comment (leightkt) [inline docs/package.json:50]:
```suggestion
    "@web3-onboard/arcana-auth": "^2.0.0-alpha.1",
```

**Comment by mmjee** [comment]
> @mmjee Overall- looking good. Don't worry about the docs not building- we can check the docs when the alpha package is published on merge into develop. I tested in the demo and was able to connect, disconnect, sign a message, and sign a typed message. However- I could not send a Goerli transaction- video of my test: https://www.loom.com/share/f95f53bfd01c404e9d816876ec1e1eb6

I tried to reproduce the above issue but I couldn't actually get it to switch to Görli because of Infura (and the API key is hardcoded).

image 1

**Comment by leightkt** [comment]
you can update the goerli rpc url in[ packages/demo/src/App.svelte](https://github.com/arcana-network/web3-onboard/blob/8ff5d621467ab250fd6cfdc7146093bbb2c5443d/packages/demo/src/App.svelte#LL251C60-L251C60)
`https://ethereum-goerli.publicnode.com`

**Comment by mmjee** [comment]
@leightkt Updated the RPC URL, it should work assuming you have Görli ETH in the address.

**Comment by leightkt** [comment]
@mmjee I still get the same 'UNSUPPORTED_OPERATION' error shown in my testing video above (able to switch to goerli, have test ETH in my account)

**Comment by leightkt** [comment]
@mmjee moving this to draft until the 'UNSUPPORTED_OPERATION' error when sending a transaction can be address. Thx!

**Comment by mmjee** [comment]
On 6/27/23 22:34, Kat Leight wrote:
> @mmjee moving this to draft until the 'UNSUPPORTED_OPERATION' error when sending a transaction can be address. Thx!
> 
Yes, it looks like we are mishandling EIP-1559 transactions so this is blocked until we figure out what the problem is and fix it.
-- 
Regards,

**Comment by mmjee** [comment]
@leightkt Can you confirm whether the issue above still occurs?

**Comment by leightkt** [comment]
@mmjee I can no longer connect to the wallet
Screenshot 2023-07-25 at 3 09 14 PM

**Comment by mmjee** [comment]
> @mmjee I can no longer connect to the wallet Screenshot 2023-07-25 at 3 09 14 PM

Updated the packages, should be working now. Works locally.

**Comment by leightkt** [comment]
Screenshot 2023-07-26 at 10 17 12 AM

**Comment by mmjee** [comment]
On ২৬/৭/২৩ ২১:৫০, Kat Leight wrote:
> Screenshot 2023-07-26 at 10 17 12 AM
> 
> 
Do you see anything in the DevTools Network Tab? This is a web wallet so
everything should be loaded in dynamically. Does that work on a browser
w/o extensions?

I can confirm it works locally.
-- 
Regards,

**Comment by leightkt** [comment]
@mmjee I was able to login only on a browser where I can no other extensions installed. Please make a note in the arcana README

Once logged in, I can switch chains, sign a message, sign a typed message, and disconnect the wallet.

I cannot send a Transaction on Goerli:
Screenshot 2023-07-27 at 3 42 08 PM

Screenshot 2023-07-27 at 3 42 11 PM

**Comment by mmjee** [comment]
On 7/28/23 03:13, Kat Leight wrote:
> @mmjee I was able to login only on a browser where I can no other extensions installed. Please make a note in the arcana README
Arcana Auth is supposed to work with all popular browsers and works even when 3rd party cookies are blocked. It should work by default for most browsers even with extensions. If you can, please narrow it down to an extension and/or configuration that could be causing it.
> 
> Once logged in, I can switch chains, sign a message, sign a typed message, and disconnect the wallet.
> 
> I cannot send a Transaction on Goerli:
> Screenshot 2023-07-27 at 3 42 08 PM
> 
> Screenshot 2023-07-27 at 3 42 11 PM
> 
> 
> 
> 
> 
Can you please retry again? We have added a few more fixes to EIP-1559 transaction handling.
-- 
Regards,

**Comment by leightkt** [comment]
@mmjee success! Please add a note to the the Arcana readme and docs page that users may need to disable other wallet extensions in order to connect to Arcana. 

Also- the Arcana docs page needs to be moved to [..1]arcana to respect alpha order. I can push that change (and handle the renaming of the other docs pages to keep the order) if you want to give me write access. 

Once those two things are completed it's good to go!

**Comment by shaloo** [comment]
> @mmjee success! Please add a note to the the Arcana readme and docs page that users may need to disable other wallet extensions in order to connect to Arcana.
> 
> Also- the Arcana docs page needs to be moved to [..1]arcana to respect alpha order. I can push that change (and handle the renaming of the other docs pages to keep the order) if you want to give me write access.
> 
> Once those two things are completed it's good to go!

Got it, fixing docs and readme as suggested above. cc: @mmjee  @leightkt

**Comment by mmjee** [comment]
@leightkt All changes are ready to be reviewed.

**Review by leightkt — APPROVED** [review]

**Comment by leightkt** [comment]
@mmjee getting two errors trying to build the package on circleci: 

Screenshot 2023-08-02 at 11 15 33 AM

**Comment by leightkt** [comment]
@mmjee getting an error on our circleci build- please put up a new PR to fix: 

Screenshot 2023-08-02 at 11 48 58 AM

**Comment by mmjee** [comment]
@leightkt Should be fixed by #1860.

**Pull request merged**
